sdk/communication/AzureCommunicationChat/Source/Options/AzureCommunicationChatClientOptions.swift

```diff
@@ -26,6 +26,11 @@ public struct AzureCommunicationChatClientOptions: AzureClientOptions {
     public let transportOptions: TransportOptions
     /// The default dispatch queue on which to call all completion handler. Defaults to `DispatchQueue.main`.
     public let dispatchQueue: DispatchQueue?
+    /// An array of `PipelineStage` policies to use in lieu of the default ones.
+    public let pipeline: [PipelineStage]?
```

Comment:
Unclear whether this is what the arch board intended, or whether they were asking for some way to inject additional policies at the end of the pipeline instead. I thought I had noted that it was the latter but I could be wrong. I think that might make sense regardless since a policy at the end of the pipeline could pretty much overwrite anything done by any earlier policy.

Reply:
This also makes sense within the context of distinguishing between "per request" and "per retry" policies.